cli/flox-rust-sdk/src/models/environment/mod.rs

```diff
@@ -459,6 +463,10 @@ pub enum EnvironmentError {
 
     #[error("could not read manifest")]
     ReadManifest(#[source] std::io::Error),
+    #[error("Failed parsing contents of manifest")]
+    ParseManifest(#[source] toml_edit::de::Error),
```

Comment:
nit
```suggestion
    // Should be unreachable since we control the template
    #[error("Failed parsing contents of template manifest")]
    ParseTemplateManifest(#[source] toml_edit::de::Error),
```